lib/lotus/generators/application/container.rb

```diff
@@ -37,6 +37,7 @@ def start
             'Gemfile.tt'               => 'Gemfile',
             'config.ru.tt'             => 'config.ru',
             'config/environment.rb.tt' => 'config/environment.rb',
+            'config/loader.rb.tt'      => 'config/loader.rb',
```

Comment:
Maybe would be a good idea to use the same naming as Rails :x

What about `config/boot.rb`?